lib/jira/resource/attachment.rb

```diff
@@ -20,26 +20,29 @@ def self.meta(client)
       end
 
       def save!(attrs)
-        headers = { 'X-Atlassian-Token' => 'nocheck' }
-        data = { 'file' => UploadIO.new(attrs['file'], 'application/binary', attrs['file']) }
+        headers = { 'X-Atlassian-Token' => 'no-check' }
+        data = { 'file' => UploadIO.new(attrs[:file], 'application/binary', attrs[:file]) }
```

Comment:
Wouldn't this break existing code that passes the `file` parameter as a string?

Reply:
You are right! Fixed that.

Reply:
Thanks 👍 ! Using symbols is a good move for API consistency, that's what I did in #322 too.